Cargo.toml

```diff
@@ -31,7 +31,8 @@ serde = { version = "1.0.124", features = ["derive"], optional = true }
 frame-metadata = { version = "15.0.0", default-features = false, features = ["v14"] }
 thiserror = "1.0.24"
 scale-info = { version = "2.0.0", default-features = false, features = ["std"] }
-scale-decode = "0.4.0"
+scale-decode = { git = "https://github.com/paritytech/scale-decode", branch = "jsdw-decode-as-type" }
```

Comment:
I guess when the `scale-decode` PR is merged then we will publish a new release and update it here before merging?